Project.toml

```diff
@@ -28,8 +28,8 @@ ImageMagick = "1.1"
 JSServe = "1.2"
 Observables = "0.3"
 ShaderAbstractions = "0.2.1"
-StaticArrays = "0.12, 1.0"
-julia = "1.3"
+StaticArrays = "0.12, 1.0.2"
+julia = "1.5, 1.6"
```

Comment:
Was the support for older Julia versions dropped?

Reply:
@SebastianM-C I think the only problem here was with supporting Julia 1.3

Reply:
@SebastianM-C 
Currently, my only change is dropping the 1.3 support
I don't know why but it downgrades a lot of stuff

Reply:
Any other problem?